src/app/main/components/recipes-list/recipes-list.component.scss

```diff
@@ -1,7 +1,7 @@
 .recipes-container {
```

Comment:
shouldn't be recipes-list.component inside recipes-section folder?
src/app/main/components/recipes-section/recipes-list/recipes-list.component.scss & ts & html